config/stack/manifests/icon.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this is the GCP icon, can you please change this to one of the Rook icons instead? This one would probably be the best: https://github.com/rook/artwork/blob/master/logo/Rook_Icon-blue.svg

Reply:
Sure thing, was going to ask you about this :)